api/management/api.go

```diff
@@ -464,6 +464,33 @@ func ConvertUpdatableToKCUser(user UpdatableUserRepresentation) kc.UserRepresent
 	return userRep
 }
 
+func ConvertToAPIRole(role kc.RoleRepresentation) RoleRepresentation {
+	var roleRep RoleRepresentation
+	roleRep.ID = role.ID
+	roleRep.Name = role.Name
+	roleRep.Composite = role.Composite
+	roleRep.ClientRole = role.ClientRole
+	roleRep.ContainerID = role.ContainerID
+	roleRep.Description = role.Description
+	return roleRep
+}
+
+func ConvertToKCRole(role RoleRepresentation) kc.RoleRepresentation {
+	attributes := map[string][]string{
```

Comment:
I think we should not hard code this here. the aims of API is to do a conversion only, here we are adding a flag silently.

Reply:
Corrected